delete-user-data/functions/src/index.ts

```diff
@@ -24,8 +24,9 @@ import * as logs from "./logs";
 // Initialize the Firebase Admin SDK
 admin.initializeApp({
   credential: admin.credential.applicationDefault(),
-  databaseURL: `https://${config.SELECTED_DATABASE_INSTANCE}.firebaseio.com`,
-});
+  databaseURL: `https://${config.selectedDatabaseInstance}${config.selectedDatabaseLocation === "us-central1" ? ".firebaseio.com" : `.${config.selectedDatabaseLocation}.firebasedatabase.app`}`
```

Comment:
Isn't it possible that `config.selectedDatabaseLocation` isn't provided (since it's not required), and this would create the URL `undefined.firebasedatabase.app`?

Reply:
Yes that is possible. Would you suggest then not initializing it at all when you won't provide `SELECTED_DATABASE_INSTANCE `as well? We could make the SELECTED_DATABASE_LOCATION required, but if you won't provide an `selectedDatabaseInstance`(SELECTED_DATABASE_INSTANCE) that will also be undefined.

Thats why I provided extra check before making a call from RTDB to accommodate that somehow.
But making `config.selectedDatabaseLocation. ` would be useful in this case to avoid an error for sure.

Reply:
I thought it'd always be there since the param has a default value: https://github.com/firebase/extensions/pull/799/files#diff-e25bb90bce079f1ec5fc20cf7bebf942c7ec97308184bef1a78ebecf19b1cfa8R179 - I might be wrong though on what the behaviour of default does

Reply:
I think I we should use the same as for the functions LOCATION so maybe
`required: true
default: us-central1
immutable: true
`
What do you think ? Happy to hear some possible implementations.

Reply:
One other issue. https://firebase.google.com/docs/projects/locations

us-central1 | Iowa | DB_NAME.firebaseio.com
-- | -- | --
europe-west1 | Belgium | DB_NAME.europe-west1.firebasedatabase.app
asia-southeast1 | Singapore | DB_NAME.asia-southeast1.firebasedatabase.app

Singapore has it's own unique subdomain. Source: https://firebase.google.com/docs/projects/locations

Reply:
@dackers86 I think that is implemented there in the code already. If it not us-central1 the address will be correct.

Reply:
@Mike @Elliot New helper function now is added with unit tests. Screenshots are updated as well. It won't initialise now if both of the parameters won't be provided and it won't make any call neither. I kept it as optional for better readability from firebase console and be consistent with other options . There is also a sidenote saying  `(Only applicable if you provided the Realtime Database instance parameter.)`